### ISSUE TITLE
Crash when parsing some non clojure files

### ISSUE THREAD

**Issue by syl20bnr** [gfredericks/how-to-ns#8]
Hi Gary,

`how-to-ns` crashes on one of our project when attempting to parse SQL files. It does not happen with all SQL files though.

Is it possible to skip non `.clj` file ?

```
lein how-to-ns fix
Picked up JAVA_TOOL_OPTIONS: -Dapple.awt.UIElement=true
Warning: implicit middleware found: lein-shorthand.plugin/middleware
Please declare all middleware in :middleware as implicit loading is deprecated.
Warning: implicit hook found: lein-shorthand.plugin/hooks
Hooks are deprecated and will be removed in a future version.
Exception in how-to-ns when checking path/to/dests.sql
#error {
 :cause "nth not supported on this type: Symbol"
 :via
 [{:type java.lang.UnsupportedOperationException
   :message "nth not supported on this type: Symbol"
   :at [clojure.lang.RT nthFrom "RT.java" 947]}]
 :trace
 [[clojure.lang.RT nthFrom "RT.java" 947]
  [clojure.lang.RT nth "RT.java" 897]
  [com.gfredericks.how_to_ns$parse_ns_form invokeStatic "how_to_ns.clj" 13]
  [com.gfredericks.how_to_ns$parse_ns_form invoke "how_to_ns.clj" 13]
  [com.gfredericks.how_to_ns$print_ns_form invokeStatic "how_to_ns.clj" 129]
  [com.gfredericks.how_to_ns$print_ns_form invoke "how_to_ns.clj" 126]
  [com.gfredericks.how_to_ns$reformat_ns_str invokeStatic "how_to_ns.clj" 231]
  [com.gfredericks.how_to_ns$reformat_ns_str invoke "how_to_ns.clj" 227]
  [com.gfredericks.how_to_ns$format_ns_str invokeStatic "how_to_ns.clj" 237]
  [com.gfredericks.how_to_ns$format_ns_str invoke "how_to_ns.clj" 233]
  [com.gfredericks.how_to_ns$format_initial_ns_str invokeStatic "how_to_ns.clj" 256]
  [com.gfredericks.how_to_ns$format_initial_ns_str invoke "how_to_ns.clj" 251]
  [com.gfredericks.how_to_ns.main$fix$fn__792 invoke "main.clj" 81]
  [com.gfredericks.how_to_ns.main$fix invokeStatic "main.clj" 79]
  [com.gfredericks.how_to_ns.main$fix invoke "main.clj" 76]
  [leiningen.how_to_ns$eval804$how_to_ns__806 doInvoke "how_to_ns.clj" 33]
  [clojure.lang.RestFn invoke "RestFn.java" 423]
  [clojure.lang.Var invoke "Var.java" 383]
  [clojure.lang.AFn applyToHelper "AFn.java" 156]
  [clojure.lang.Var applyTo "Var.java" 700]
  [clojure.core$apply invokeStatic "core.clj" 648]
  [clojure.core$apply invoke "core.clj" 641]
  [leiningen.core.main$partial_task$fn__4687 doInvoke "main.clj" 284]
  [clojure.lang.RestFn applyTo "RestFn.java" 139]
  [clojure.lang.AFunction$1 doInvoke "AFunction.java" 29]
  [clojure.lang.RestFn applyTo "RestFn.java" 137]
  [clojure.core$apply invokeStatic "core.clj" 648]
  [clojure.core$apply invoke "core.clj" 641]
  [leiningen.core.main$apply_task invokeStatic "main.clj" 334]
  [leiningen.core.main$apply_task invoke "main.clj" 320]
  [leiningen.core.main$resolve_and_apply invokeStatic "main.clj" 340]
  [leiningen.core.main$resolve_and_apply invoke "main.clj" 336]
  [leiningen.core.main$_main$fn__4760 invoke "main.clj" 449]
  [leiningen.core.main$_main invokeStatic "main.clj" 439]
  [leiningen.core.main$_main doInvoke "main.clj" 436]
  [clojure.lang.RestFn invoke "RestFn.java" 421]
  [clojure.lang.Var invoke "Var.java" 383]
  [clojure.lang.AFn applyToHelper "AFn.java" 156]
  [clojure.lang.Var applyTo "Var.java" 700]
  [clojure.core$apply invokeStatic "core.clj" 646]
  [clojure.main$main_opt invokeStatic "main.clj" 314]
  [clojure.main$main_opt invoke "main.clj" 310]
  [clojure.main$main invokeStatic "main.clj" 421]
  [clojure.main$main doInvoke "main.clj" 384]
  [clojure.lang.RestFn invoke "RestFn.java" 457]
  [clojure.lang.Var invoke "Var.java" 394]
  [clojure.lang.AFn applyToHelper "AFn.java" 165]
  [clojure.lang.Var applyTo "Var.java" 700]
  [clojure.main main "main.java" 37]]}
java.lang.UnsupportedOperationException: nth not supported on this type: Symbol
 at clojure.lang.RT.nthFrom (RT.java:947)
    clojure.lang.RT.nth (RT.java:897)
    com.gfredericks.how_to_ns$parse_ns_form.invokeStatic (how_to_ns.clj:13)
    com.gfredericks.how_to_ns$parse_ns_form.invoke (how_to_ns.clj:13)
    com.gfredericks.how_to_ns$print_ns_form.invokeStatic (how_to_ns.clj:129)
    com.gfredericks.how_to_ns$print_ns_form.invoke (how_to_ns.clj:126)
    com.gfredericks.how_to_ns$reformat_ns_str.invokeStatic (how_to_ns.clj:231)
    com.gfredericks.how_to_ns$reformat_ns_str.invoke (how_to_ns.clj:227)
    com.gfredericks.how_to_ns$format_ns_str.invokeStatic (how_to_ns.clj:237)
    com.gfredericks.how_to_ns$format_ns_str.invoke (how_to_ns.clj:233)
    com.gfredericks.how_to_ns$format_initial_ns_str.invokeStatic (how_to_ns.clj:256)
    com.gfredericks.how_to_ns$format_initial_ns_str.invoke (how_to_ns.clj:251)
    com.gfredericks.how_to_ns.main$fix$fn__792.invoke (main.clj:81)
    com.gfredericks.how_to_ns.main$fix.invokeStatic (main.clj:79)
    com.gfredericks.how_to_ns.main$fix.invoke (main.clj:76)
    leiningen.how_to_ns$eval804$how_to_ns__806.doInvoke (how_to_ns.clj:33)
    clojure.lang.RestFn.invoke (RestFn.java:423)
    clojure.lang.Var.invoke (Var.java:383)
    clojure.lang.AFn.applyToHelper (AFn.java:156)
    clojure.lang.Var.applyTo (Var.java:700)
    clojure.core$apply.invokeStatic (core.clj:648)
    clojure.core$apply.invoke (core.clj:641)
    leiningen.core.main$partial_task$fn__4687.doInvoke (main.clj:284)
    clojure.lang.RestFn.applyTo (RestFn.java:139)
    clojure.lang.AFunction$1.doInvoke (AFunction.java:29)
    clojure.lang.RestFn.applyTo (RestFn.java:137)
    clojure.core$apply.invokeStatic (core.clj:648)
    clojure.core$apply.invoke (core.clj:641)
    leiningen.core.main$apply_task.invokeStatic (main.clj:334)
    leiningen.core.main$apply_task.invoke (main.clj:320)
    leiningen.core.main$resolve_and_apply.invokeStatic (main.clj:340)
    leiningen.core.main$resolve_and_apply.invoke (main.clj:336)
    leiningen.core.main$_main$fn__4760.invoke (main.clj:449)
    leiningen.core.main$_main.invokeStatic (main.clj:439)
    leiningen.core.main$_main.doInvoke (main.clj:436)
    clojure.lang.RestFn.invoke (RestFn.java:421)
    clojure.lang.Var.invoke (Var.java:383)
    clojure.lang.AFn.applyToHelper (AFn.java:156)
    clojure.lang.Var.applyTo (Var.java:700)
    clojure.core$apply.invokeStatic (core.clj:646)
    clojure.main$main_opt.invokeStatic (main.clj:314)
    clojure.main$main_opt.invoke (main.clj:310)
    clojure.main$main.invokeStatic (main.clj:421)
    clojure.main$main.doInvoke (main.clj:384)
    clojure.lang.RestFn.invoke (RestFn.java:457)
    clojure.lang.Var.invoke (Var.java:394)
    clojure.lang.AFn.applyToHelper (AFn.java:165)
    clojure.lang.Var.applyTo (Var.java:700)
    clojure.main.main (main.java:37)
```

Thank you.

**Comment by gfredericks** [comment]
I wouldn't mind this being fixed, but have you considered putting those sql files in the `/resources` directory instead?

**Comment by danielcompton** [comment]
I get a similar error from the `.DS_Store` files that macOS creates:

```clojure
$ lein how-to-ns fix
Tried to load commons-io version 2.5 but 2.6 was already loaded.
Exception in how-to-ns when checking src/.DS_Store
#error {
 :cause "nth not supported on this type: Symbol"
 :via
 [{:type java.lang.UnsupportedOperationException
   :message "nth not supported on this type: Symbol"
   :at [clojure.lang.RT nthFrom "RT.java" 947]}]
 :trace
```

**Comment by gfredericks** [comment]
I will aim to fix this next week.

**Comment by gfredericks** [comment]
This has been released as version `0.2.3`
